ONSdigital/ssdc-rm-ddl
# Add fulfilment permissions to support group

# Motivation and Context
The RM Support Actions group doesn't have the permission to be able to request a fulfilment against a case.  You could only do this as a super user.


# What has changed
Added the permissions to request a fulfilment against a case.

# How to test?
-  In the support tool, add yourself to the RM Support Actions group and remove any super user access you might have.  Go into a case and you should see the 3 options for requesting a fulfilment for the case (email, SMS or print).
 
# Links
https://trello.com/c/tquMwC6i
